[2025-07-15T03:34:51.270+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T03:34:51.301+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflights_etl_pipeline.transform manual__2025-07-15T03:34:47.679067+00:00 [queued]>
[2025-07-15T03:34:51.310+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflights_etl_pipeline.transform manual__2025-07-15T03:34:47.679067+00:00 [queued]>
[2025-07-15T03:34:51.344+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-15T03:34:51.362+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform> on 2025-07-15 03:34:47.679067+00:00
[2025-07-15T03:34:51.372+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1455) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-15T03:34:51.371+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'airflights_etl_pipeline', 'transform', 'manual__2025-07-15T03:34:47.679067+00:00', '--job-id', '1669', '--raw', '--subdir', 'DAGS_FOLDER/airflight_pipeline.py', '--cfg-path', '/tmp/tmpk3sg2eb4']
[2025-07-15T03:34:51.374+0000] {standard_task_runner.py:63} INFO - Started process 1457 to run task
[2025-07-15T03:34:51.374+0000] {standard_task_runner.py:91} INFO - Job 1669: Subtask transform
[2025-07-15T03:34:51.391+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-15T03:34:51.425+0000] {task_command.py:426} INFO - Running <TaskInstance: airflights_etl_pipeline.transform manual__2025-07-15T03:34:47.679067+00:00 [running]> on host 5a1293588d06
[2025-07-15T03:34:51.672+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='airflights_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T03:34:47.679067+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T03:34:47.679067+00:00'
[2025-07-15T03:34:51.674+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T03:34:51.685+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T03:34:51.686+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: transform_etl() missing 1 required positional argument: 'output_file_full'
[2025-07-15T03:34:51.695+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=airflights_etl_pipeline, task_id=transform, run_id=manual__2025-07-15T03:34:47.679067+00:00, execution_date=20250715T033447, start_date=20250715T033451, end_date=20250715T033451
[2025-07-15T03:34:51.707+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 1669 for task transform (transform_etl() missing 1 required positional argument: 'output_file_full'; 1457)
[2025-07-15T03:34:51.748+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-15T03:34:51.771+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T03:34:51.809+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
